test/e2e/functional/dag-disable-failFast.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Please move this file under test/e2e/expectedfailures

Reply:
OK